.bash_profile

```diff
@@ -28,7 +28,7 @@ SRC_DIR="${HOME}/reopt_api/reo/src"
 #------------------------------------------------------------------------------
 #  Choose your solver
 #------------------------------------------------------------------------------
-export SOLVER="cbc"
+export SOLVER="xpress"
```

Comment:
we set this to cbc so that the file would work for users that do not have an Xpress license. please keep it at cbc.

Reply:
This has been updated in commit 5521c5e3384e4dfb46eb543cda3a9e0e961a85bd